package/MDAnalysis/core/selection.py

```diff
@@ -1353,6 +1353,51 @@ def _apply(self, group):
         return group[mask]
 
 
+class RelPropertySelection(PropertySelection):
+    """Some of the possible properties:
+    x, y, z,
+
+    .. versionadded:: 2.9.0
+    """
+
+    token = "relprop"
+    precedence = 1
+
+    def __init__(self, parser, tokens):
+        super().__init__(parser, tokens)
+        self.sel = parser.parse_expression(self.precedence)
+        # self.ori_value = self.value
+
+    def _apply(self, group):
+        try:
+            values = getattr(group, self.props[self.prop])
+        except KeyError:
+            errmsg = f"Expected one of {list(self.props.keys())}"
+            raise SelectionError(errmsg) from None
+        except NoDataError:
+            attr = self.props[self.prop]
+            errmsg = f"This Universe does not contain {attr} information"
+            raise SelectionError(errmsg) from None
+
+        try:
+            col = {"x": 0, "y": 1, "z": 2}[self.prop]
+        except KeyError:
+            pass
+        else:
+            values = values[:, col]
+            sel = self.sel.apply(group)
+            rel_value = (
+                sel.center_of_geometry().reshape(3).astype(np.float32)[col]
```

Comment:
Your docs should make clear that the COM of the second group is used.

Is this what this selection should do? Or should it calculate all-vs-all distances and then compute the min over all distances and then compare this minimal distance to the value?

Reply:
When using [center_of_geometry](https://docs.mdanalysis.org/dev/documentation_pages/core/groups.html#MDAnalysis.core.groups.AtomGroup.center_of_geometry) you also need to think carefully to what value `wrap` needs to be set, especially if the user does not get a choice in setting it.

Reply:
Yes. GOM (not COM) is what I want to implement. In my cases, the planar surfaces are used, and taking the average positions of the surfaces is better than the minimal distances as reference. But I can see the case when the minimal distance would be a better choice, for example, for the curved surfaces. I suggest to have a new selecton class. What is your opinion?